src/ui/profile_preferences.ui

```diff
@@ -1197,6 +1208,9 @@
           </item>
           <item row="2" column="1">
            <widget class="QPushButton" name="pushButton_command_foreground_color">
+            <property name="toolTip">
+             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The foreground color used in the main console when text sent to the MUD Server, or which was processed locally, which was entered in the command line, is shown afterward (when the &amp;quot;&lt;span style=&quot; font-style:italic;&quot;&gt;Show the text you sent&lt;/span&gt;&amp;quot; option is checked).&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
```

Comment:
> The foreground color used for the command you've sent to the game

Reply:
> The foreground color used for text you have entered and sent.

Reply:
I'm not sure the word "entered" should be used as it can be associated with the "command line" part where the text is entered.  Indeed, I think we have to mention that this setting will not have any effect if the `Show the text you sent` setting is not checked so as to draw a clear difference between the two sets of similarly named controls.

Reply:
> The foreground color of text you have entered and sent.

Reply:
If you deactivate to show sent text, then you won't see any and it needs no color. I think this would be clear. 
At some point I'd like more extensive explanation of different option windows, etc. Maybe a small question mark icon, but tooltips should be concise, no?

Reply:
I think we're over-engineering it here. Do we have evidence of people being confused by this? It has not been a pain point in Mudlet.

Reply:
I agree to both! :smile:

Reply:
Will change to "The {fore|back}ground color used for text sent to the game server." in pending commit